### DISCUSSION TITLE
Census II of Free and Open Source Software — Application Libraries

### DISCUSSION THREAD

**Discussion by wangleo61** [kaiyuanshe/ONES#56] [资料分享]
Census II of Free and Open Source Software —application Libraries

有兴趣的小伙伴一起来看看LF 和OpenSSF的最新的研究报告
[LFResearch_Harvard_Census_II.pdf](https://github.com/kaiyuanshe/ONES/files/8305379/LFResearch_Harvard_Census_II.pdf)
 Software Bill of Materials (SBOM) and Cybersecurity Readiness
[LFResearch_SBOM_Report_020422.pdf](https://github.com/kaiyuanshe/ONES/files/8305450/LFResearch_SBOM_Report_020422.pdf)
Report on the2020 FOSS Contributor Survey
[2020FOSSContributorSurveyReport_121020.pdf](https://github.com/kaiyuanshe/ONES/files/8305465/2020FOSSContributorSurveyReport_121020.pdf)
Vulnerabilities in the Core Preliminary Report and Census II of Open Source Software
[census_ii_vulnerabilities_in_the_core.pdf](https://github.com/kaiyuanshe/ONES/files/8305495/census_ii_vulnerabilities_in_the_core.pdf)
Improving Trust and Security in Open Source Projects
[improving_trust_security_in_oss_projects.pdf](https://github.com/kaiyuanshe/ONES/files/8305525/improving_trust_security_in_oss_projects.pdf)

**Comment by wangleo61** [comment]
有兴趣的小伙伴一起来看看LF 和OpenSSF的最新的研究报告
[LFResearch_Harvard_Census_II.pdf]
启示录
1 The Need for a Standardized Naming Schema for Software Components
2 The Complexities Associated with Package Versions
3 Much of the Most Widely Used FOSS Is Developed by Only a Handful of Contributors
4 The Increasing Importance of Individual Developer Account Security
5 The Persistence of Legacy Software in the Open Source Space

**Comment by will-ww** [comment]
这个好，刚打印了一份出来看，找个机会一起分享下~

**Reply by wangleo61** [reply]
好的
